qtodotxt/ui/controllers/main_controller.py

```diff
@@ -404,8 +411,8 @@ def updateFilters(self):
         self._onFilterSelectionChanged(self._filters_tree_controller.view.getSelectedFilters())
 
     def toggleVisible(self):
-        if self.view.isMinimized():
+        if self.view.isMinimized() or self.view.isHidden():
             self.view.showNormal()
```

Comment:
Not quite correct behavior in the sense that if the window is maximum, after minimize and restore from tray, the window will have normal size